src/state/reducers/scigateway.reducer.test.tsx

```diff
@@ -261,6 +261,24 @@ describe('scigateway reducer', () => {
     });
   });
 
+  it('dismissNotification should remove the referenced notification from the notifications list in State', () => {
```

Comment:
The name of this test doesn't reflect what it's testing - presumably it should be more along the lines of "new notification isn't added to notifications if it is a duplicate" etc.